tests/ert3/engine/integration/test_engine.py

```diff
@@ -0,0 +1,49 @@
+import pathlib
+import os
+import sys
+import unittest
+import shutil
+
+import pytest
+
+import ert3
+from ert3.console._console import _load_ensemble_config, _load_stages_config
+
+_EXAMPLES_ROOT = (
+    pathlib.Path(os.path.dirname(__file__)) / ".." / ".." / ".." / ".." / "examples"
+)
+_POLY_WORKSPACE_NAME = "polynomial"
+_POLY_WORKSPACE = _EXAMPLES_ROOT / _POLY_WORKSPACE_NAME
```

Comment:
Have you considered providing these files in another way? Like attaching some test data to the test, or providing them via fixtures?

Reply:
Yes, will do that, this was just copying the existing tests set up. It is still relying a bit too much on files in magical locations...